lib/image-inspector-client.rb

```diff
@@ -21,10 +21,12 @@ def initialize(
         password:          nil,
         bearer_token:      nil
       },
-      http_proxy_uri:   nil
+      http_proxy_uri:   nil,
+      auth_token: nil
```

Comment:
rubocop seems to hate it when we have more than 5 arguments. We should either change its definitions of maybe group `http_proxy_uri` and `auth_token` under a general `options` argument? or even better put `auth_token` into `auth_options`.